examples/fscloud/variables.tf

```diff
@@ -57,3 +57,9 @@ variable "allow_vpcs_to_cos" {
   description = "Set rule for VPCs to COS, deafult is true"
   default     = true
 }
+
+variable "skip_specific_services_for_zone_creation" {
```

Comment:
I'd suggest to add the default values inline in the main as opposed to a variable, along with some comment indicating that this is for illustration purposes ( we're moving away from having input variables in the examples )

Reply:
done